gematria/datasets/pipelines/compile_modules_lib_test.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+import os
+
+from gematria.datasets.pipelines import compile_modules_lib
+from gematria.testing.python import ir_utils
+
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
```

Comment:
The Google style guide does not allow direct imports of classes or functions (with the exception of `collections.abc` and `typing`).

See https://google.github.io/styleguide/pyguide.html#22-imports for more thoughts on this.